.github/workflows/ci.yml

```diff
@@ -106,6 +106,25 @@ jobs:
           git commit --reset-author --amend -m "Update from GitHub Actions"
           git push --force origin gh-pages
 
+  copyright_check:
+    runs-on: [ubuntu-20.04]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Run check
+        run: |
+          sed -i \
+            -e "/Copyright (c).*-.* Intel Corporation/s/[0-9]\+ /$(date +%Y) /" \
+            -e "/Copyright (c) [0-9]\+ Intel Corporation/s/[0-9]\+/&-$(date +%Y)/" \
```

Comment:
```suggestion
            -e "/Copyright (c) .* Intel Corporation/s/ \([0-9]\+\)[-0-9]*/ \\1-$(date +%Y)/" \
            -e "/Copyright (c) .* Intel Corporation/s/$(date +%Y)-$(date +%Y)/$(date +%Y)/" \
```

Reply:
This code solves the problem in case of correct year. I applied this change.